src/main/resources/static/public/style.css

```diff
@@ -0,0 +1,39 @@
+#container {
```

Comment:
Why is this file needed? I don't see any web user interface in this application.

Reply:
> Why is this file needed? I don't see any web user interface in this application.

Initially for adding payments/cases I created two simple POST forms using Thymeleaf view renderer. But later I made API fully RESTful (only JSON for requests), so forms wouldn't work. Then removed them, but forgot about style :)